std/algorithm/comparison.d

```diff
@@ -560,7 +560,7 @@ if (isInputRange!R1 && isInputRange!R2 && !(isSomeString!R1 && isSomeString!R2))
     }
 }
 
-// Specialization for strings (for speed purposes)
+/// Specialization for strings (for speed purposes)
```

Comment:
Like this it will turn up in its own section... with a really sparse description. That would look pretty bad...